README.md

```diff
@@ -20,6 +20,66 @@ Fork this repository and add your tables here:
 
 (your tables go here - you can use TableGenerator to create them. Include the data from the sample above in your tables)
 
+### Authors
+
+| AuthorID | FullName      | Email                       | Bio            |
+|----------|---------------|-----------------------------|----------------|
+| 1        | Eric Fleming  | ericfleming@nimblepros.com  | Long Text Here |
+| 2        | Kyle McMaster | kylemcmaster@nimblepros.com | Long Text Here |
+| 3        | Steve Smith   | steve@nimblepros.com        | Long Text Here |
+
+### BlogPosts
+
+| Slug                          | Title                                | Body           | AuthorID | Views |
+|-------------------------------|--------------------------------------|----------------|----------|-------|
+| serilog-structured-logging    | Structured Logging with Serilog      | Long Text Here | 1        | 123   |
+| persisting-smart-enum         | Persisting a Smart Enum with EF Core | Long Text Here | 2        | 234   |
+| localization-aspnet-core-apis | Localization in ASP.NET Core APIs    | Long Text Here | 3        | 345   |
+
+### Categories
+
+| CategoryID | Category             |
+|------------|----------------------|
+| 1          | Software Development |
+| 2          | Cloud Computing      |
+| 3          | Web APIs             |
```

Comment:
Looks good.

BTW, my personal preference is to just use `Id` as my Id column. There are pros and cons to using `Id` vs. `TableNameId` but in my opinion the tradeoffs end in favor of Id. Here's a quick summary:
Pros for Id:
Id is shorter.
Id is less redundant
Id maps better to classes that have an Id property
Using an Id property as your standard on classes lends itself to base classes, polymorphism, and generics

Pros for TableNameId
When doing joins it's more clear which id is which without having to use aliases

Reply:
Sounds good, I will accept the suggestion!